src/smart-components/overview/overview.js

```diff
@@ -34,13 +34,17 @@ import './overview.scss';
 import { Table, Tbody, Td, Tr } from '@patternfly/react-table';
 import AppLink from '../../presentational-components/shared/AppLink';
 import pathnames from '../../utilities/pathnames';
+import EnableWorkspacesAlert from './enable-workspaces-alert';
+import { useFlag } from '@unleash/proxy-client-react';
 
 const Overview = () => {
   const intl = useIntl();
   const [expanded, setExpanded] = useState(true);
+  const isWorkspacesFlag = useFlag('platform.rbac.workspaces');
 
   return (
     <React.Fragment>
+      {!isWorkspacesFlag && <EnableWorkspacesAlert />}
```

Comment:
This is probably not what we want - it will show the alert to every user (even in production) since the flag will return false. This should also appear only if the flag has a true value.

Reply:
oops I forgot to flip it after testing